front/lib/api/llm/clients/anthropic/utils/test/conversation_to_anthropic.test.ts

```diff
@@ -0,0 +1,102 @@
+import { describe, expect, it } from "vitest";
+
+import { toMessage } from "@app/lib/api/llm/clients/anthropic/utils/conversation_to_anthropic";
+import type { ModelMessageTypeMultiActionsWithoutContentFragment } from "@app/types";
+
+describe("toMessage", () => {
+  describe("user messages", () => {
+    it("should convert user message with text and function calls.", () => {
+      const messages = conversationMessages.map(toMessage);
+
+      expect(messages).toEqual(inputMessages);
+    });
+  });
+});
+
+const conversationMessages: ModelMessageTypeMultiActionsWithoutContentFragment[] =
+  [
+    {
+      role: "user",
+      name: "Pierre Milliotte",
```

Comment:
You can update this to "Somebody" or anything, oversight from my end in Mistral PR